app/_locales/en/messages.json

```diff
@@ -3374,8 +3374,8 @@
     "description": "The description of the `endowment:network-access` permission."
   },
   "permission_accessNetworkDescription": {
-    "message": "Allow the snap to access the internet. This can be used to both send and receive data with third-party servers.",
```

Comment:
We should delete all the translated strings for all of these to have them retranslated